docs/getting_started/basics/ui_overview.md

```diff
@@ -1,63 +1,124 @@
 
 ## Board
 
-Right under the toolbar you will find the Shift board. This element is what will display your workflow graph. You can interact with it to create nodes, connect them together and navigate your creation.  
+The Shift board is the main workspace for creating the workflow graph. On the board it is possible to create and interact with nodes, link them, and navigate the graph.
+
+The following controls are available for the board:
+
+|Action|Short-Cut|
+|:--|:--|
+|*Pan*|`Alt + Left-Click` or `Middle-Click`|
+|*Zoom*|`Alt + Right-Click` or `Mouse Wheel`|
+|*Focus on Selected*|`F`|
+|*Select*|`Left-Click`|
+|*Add to Selection*|`Shift + Left-Click`|
+|*Create Node*|`Double Left-Click` or `Tab`|
+|*Delete Selected*|`Del`|
+|*Rectangular Selection*|`Left-Click and Drag`|
+
+Several Shift boards can be opened to work on different tasks at the same time; each will be displayed as a separate tab.
+
+## Dockable Widgets
+All other widgets in the Shift Window are utility widgets used to interact with the workflow and the application. These widgets are dockable widgets, which means that they can be resized and rearranged throughout the Shift window to satisfy personal layout tastes.
+New dockable widgets can also be added through [plugins](../../reference/plugins.md); these additional widgets can be added and removed from the UI when needed.
 
-**Controls**:
+Shift standalone comes with three native and indispensable widgets:
 
-- "*Pan*": `Alt Key + Left-Click` or `Middle-Click`
-- "*Zoom*": `Alt Key + Right-Click` or `Mouse Wheel`
-- "*Focus on Selected*": `f Key`
-- "*Select*": `Left-Click`
-- "*Add to Selection*": `Shift Key + Left-Click`
-- "*Create Node*": `Double Left-Click` or `Tab Key`
-- "*Delete Selected*": `Delete Key`
-- "*Recatangular Selection*": `Left-Click and Drag`
+### The Inspector
 
-You can open as many Shift boards as you want to work on different tasks at the same time; each will be displayed as a separate tab.
+The inspector is used to display and modify the properties of the selected nodes in the workflow. It is divided into four sections (from top to bottom):
 
-## Inspector
+- The node's unique name.
+- The parameters group: It contains the operator's name, which can be modify to change the name of the node.
+- The node's inputs: List of node inputs and its corresponding values. Input values can be modified here as long as they are not connected to another plug; if this is the case, the corresponding widget will be disabled and greyed out.
+- The node's outputs: List of node outputs and its corresponding values.
 
-It is used to display and modify the properties of the operators in the workflow. If we select an operator, the inspector will react displaying its properties.
-The inspector is a component of the Shift window whose location and size can be customized. In our example figures it is located on the right of the main window.
+<figure>
```

Comment:
In this image, still appears "Operator Name", we should keep track and change it once the Shift PR (https://github.com/Inbibo/shift/pull/1113) is merged to update the image.

Reply:
I will change it, thanks for pointing it out!